Don't enable event collection by default on OSP

leifmadsen opened the pull request:
Closes STF-1498

csibbitt reviewed (approved):
LGTM as a first pass

vkmc commented on doc-Service-Telemetry-Framework/modules/proc_creating-the-base-configuration-for-director-operator-for-stf.adoc:
```diff
@@ -41,7 +41,7 @@ data:
         # enable Ceilometer metrics and events
         CeilometerQdrPublishMetrics: true
-        CeilometerQdrPublishEvents: true
+        CeilometerQdrPublishEvents: false
```
We could drop this config, the default value is false. See https://opendev.org/openstack/tripleo-heat-templates/src/branch/master/deployment/ceilometer/ceilometer-agent-notification-container-puppet.yaml#L53-L56

leifmadsen replied:
OK thanks! I was looking at that but hadn't gotten around to looking up the defaults.

vkmc commented on doc-Service-Telemetry-Framework/modules/proc_creating-the-base-configuration-for-stf.adoc:
```diff
@@ -28,9 +28,9 @@ parameter_defaults:
+    # enable Ceilometer metrics
     CeilometerQdrPublishMetrics: true
-    CeilometerQdrPublishEvents: true
+    CeilometerQdrPublishEvents: false
```
Ditto

vkmc commented on doc-Service-Telemetry-Framework/modules/proc_creating-the-base-configuration-for-stf.adoc:
```diff
@@ -104,9 +104,9 @@ parameter_defaults:
+    # enable Ceilometer metrics
     CeilometerQdrPublishMetrics: true
-    CeilometerQdrPublishEvents: true
+    CeilometerQdrPublishEvents: false
```
Ditto

vkmc reviewed (approved):
Looks good to me. I think we could just delete the option from the environment file altogether and that would be enough, but if we want to be explicit with events not being enabled I would leave it as is.

leifmadsen commented:
I have updated everything after double checking the defaults Victoria pointed out along with the default Events publishing list (which is also empty). I've removed the extra items from the configuration now.
